modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/utils/utils.es.js

```diff
@@ -187,4 +188,17 @@ export function dateToInternationalHuman(
 	const intl = new Intl.DateTimeFormat(localeKey, options);
 
 	return intl.format(date);
+}
+
+/**
+ * Returns a YYYY-MM-DD date
+ * based on a JS Date object
+ *
+ * @export
+ * @param {Date} dateJsObject
+ * @returns {string}
+ */
+export function jsDatetoDate(dateJsObject) {
```

Comment:
The more verbose the better, even something like `jsDateToYYYYMMDD`.

Or you could pass in the format as a second argument and call it `jsDateToDateString(dateObject, format = 'YYYY-MM-DD')` where the format is defaulted to that style

Reply:
Thanks! That name wasn't great. I went for `jsDateToYYYYMMDD`, as the function is not meant to be flexible. Even though I'm not completely happy because the name doesn't express the actual format it returns.